### PR TITLE
explain images in cc are same as md/mdx images

### PR DESCRIPTION
**Pull request by OliverSpeir**
#### Description (required)

Adds explanation that md/mdx images are treated the same in CC as normal md/mdx

### CODE REVIEW

**Review by at-the-vr — APPROVED** [review]
LGTM 👍

File: src/content/docs/en/guides/images.mdx [hunk 1]
```diff
@@ -518,7 +518,9 @@ import rocket from '../assets/rocket.png';
 
 ## Images in Content Collections
 
-You can declare an associated image for a content collections entry, such as a blog post's cover image, in your frontmatter using its path relative to the current folder:
+Images in Content Collections will be processed the same way they are in [Markdown](#images-in-markdown-files) and [MDX](#images-in-mdx-files) depending on which filetype you are using.
```
Comment (alexnguyennz) [inline src/content/docs/en/guides/images.mdx:521]:
```suggestion
Images in Content Collections will be processed the same way they are in [Markdown](#images-in-markdown-files) and [MDX](#images-in-mdx-files) depending on which file type you are using.
```

Comment (alexnguyennz) [reply src/content/docs/en/guides/images.mdx:521]: Since the docs tend to use file type

File: src/content/docs/en/guides/images.mdx [hunk 2]
```diff
@@ -518,7 +518,9 @@ import rocket from '../assets/rocket.png';
 
 ## Images in Content Collections
 
-You can declare an associated image for a content collections entry, such as a blog post's cover image, in your frontmatter using its path relative to the current folder:
+Images in Content Collections will be processed the same way they are in [Markdown](#images-in-markdown-files) and [MDX](#images-in-mdx-files) depending on which file type you are using.
+
+Additionally you can declare an associated image for a content collections entry, such as a blog post's cover image, in your frontmatter using its path relative to the current folder:
```
Comment (sarah11918) [inline src/content/docs/en/guides/images.mdx:523]:
```suggestion
Additionally, you can declare an associated image for a content collections entry, such as a blog post's cover image, in your frontmatter using its path relative to the current folder:
```

File: src/content/docs/en/guides/images.mdx [hunk 3]
```diff
@@ -518,7 +518,9 @@ import rocket from '../assets/rocket.png';
 
 ## Images in Content Collections
```
Comment (sarah11918) [inline src/content/docs/en/guides/images.mdx:519]:
```suggestion
## Images in content collections
```

File: src/content/docs/en/guides/images.mdx [hunk 4]
```diff
@@ -518,7 +518,9 @@ import rocket from '../assets/rocket.png';
...
 
-You can declare an associated image for a content collections entry, such as a blog post's cover image, in your frontmatter using its path relative to the current folder:
+Images in Content Collections will be processed the same way they are in [Markdown](#images-in-markdown-files) and [MDX](#images-in-mdx-files) depending on which file type you are using.
```
Comment (sarah11918) [inline src/content/docs/en/guides/images.mdx:521]:
```suggestion
Images in content collections will be processed the same way they are in [Markdown](#images-in-markdown-files) and [MDX](#images-in-mdx-files) depending on which file type you are using.
```

**Review by sarah11918 — COMMENTED** [review]
Lovely, great to have!

**Pull request merged**